Installation/CHANGES.md

```diff
@@ -41,6 +41,14 @@ Release date: December 2021
 
 -   Added documentation for the class `Projection_traits_3`, which enables the use of 2D algorithms on the projections of 3D data onto an arbitrary plane.
 
+-   Most operations on `Exact_predicates_exact_constructions_kernel` objects are now thread-safe
+    if `CGAL::Exact_rational` is `mpq_class` (from `GMPXX`), `boost::multiprecision::mpq_rational` or
+    `CGAL::Quotient<CGAL::MP_Float>`. The objects are not atomic though, so the usual restrictions
+    on avoiding race conditions apply. For users who do not use threads, this can be disabled with `CGAL_HAS_NO_THREADS`.
```

Comment:
This paragraph will probably be obscure for most users. We should restrict to the default for `Exact_predicates_exact_constructions_kernel`. Users do not really know or care what is the number type that is used to implement the kernel.

Reply:
This could be left as is and a note added in `General` referring to here for more details